ansible-report/packaging/prerm

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -x
+
+# Import Ansible Technique
+NAME=ansible_report
+FOLDERS="ncf/50_techniques techniques dsc/ncf/50_techniques techniques/ncf_techniques"
+cd /var/rudder/configuration-repository/
+git reset
+for folder in $FOLDERS
+do
+  git rm -r $folder/$NAME
+done
+git commit -m "Ansible-report plugin uninstallation"
+# need to be run 2 times to be updated, first one will fail
```

Comment:
Do you have any idea why?

Reply:
Not really the error given is the following:

+ rudder server reload-techniques
error: Could not reload techniques from files
Error: An error occured when updating the Technique library from file system
cause:Error when trying to delete category with ID 'ActiveTechniqueCategoryId(ansible_report)'
cause:Can not delete 'techniqueCategoryId=ansible_report,techniqueCategoryId=Active Techniques,ou=Rudder,cn=rudder-configuration': subordinate objects must be deleted first

Reply:
Ok, could you open a ticket and add the link in a comment with a TODO to remove it?